packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -547,3 +547,814 @@ declare module 'lightning/uiRecordApi' {
+    }
+
+    /**
+     * Asset reference representation.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#AssetReferenceRepresentation
+     *
+     * Keys:
+     *    (none)
+     */
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    export interface AssetReferenceRepresentation extends BaseAssetReferenceRepresentation {}
+
+    /**
+     * Base asset reference representation.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#AssetReferenceRepresentation
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface BaseAssetReferenceRepresentation {
+        /** The 18 character ID of the asset. */
+        id: string;
+        /** The asset label. */
+        label?: string;
+        /** The asset developer name. */
+        name?: string;
+        /** The namespace that qualifies the asset name */
+        namespace?: string;
+        /** The asset URL. */
+        url?: string;
+    }
+
+    /**
+     * Base Wave asset representation.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#BaseWaveAssetRepresentation
+     * Keys:
+     *    id (string): id
+     */
+    export interface BaseWaveAssetRepresentation {
+        /** Sharing URL for the asset. */
+        assetSharingUrl?: string | null;
+        /** The user that created the asset. */
+        createdBy?: WaveUserRepresentation;
+        /** Time the asset was created. */
+        createdDate?: string;
+        /** Short description of the asset. */
+        description?: string;
+        /** The 18 character asset ID. */
+        id: string;
+        /** The label of the asset. */
+        label?: string;
+        /** Last time the asset was accessed. */
+        lastAccessedDate?: string | null;
+        /** The user that last updated the asset. */
+        lastModifiedBy?: WaveUserRepresentation | null;
+        /** Last time the asset was modified. */
+        lastModifiedDate?: string | null;
+        /** The name of the asset. */
+        name?: string;
+        /** The namespace of the Asset. */
+        namespace?: string;
+        /** Represents permissions for the present user. */
+        permissions?: PermissionsRepresentation | null;
+        /** The asset type. */
+        type: string;
+        /** URL to get the definition of the asset. */
+        url: string;
+    }
+
+    /**
+     * Daily schedule on which to execute some type of job.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#DailyScheduleInputRepresentation
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface DailyScheduleInputRepresentation extends ScheduleInputRepresentation {
+        frequency: 'daily' | 'Daily';
```

Comment:
This is a connect enum used as a discriminator in lds, which comes through as a discriminated union type. Wave connect enums are case-insensitive on input and generally come through as camel-case in output, hence this `|`ed type on the various `frequency`'s.

Reply:
I'm wondering if in an intellisense context when it shows both values it could be confusing whether there's a difference between capital or not. Does it make sense to have the docstring for the base interface's `frequency` prop to call out it is case-insensitive?

Reply:
Makes sense, done.